lib/core/app/jobs/ros/application_job.rb

```diff
@@ -20,7 +20,7 @@ class ApplicationJob < ActiveJob::Base
     end
 
     def perform(*params)
-      operation_class(*params).new.call(*params)
+      OperationResult.new(*operation_class.call(params))
```

Comment:
I think this will break the storage service b/c there is `OperationResult` in the storage service